### ISSUE TITLE
Make file doesn't work

### ISSUE THREAD

**Issue by dant1ux** [C3RV1/UndertaleNDS#37]
Hello I've followed the steps to building the .nds and I've entered make and it gives me this:
`make[1]: *** No rule to make target '/home/user/Downloads/UndertaleNDS-0.1/nitrofs/stamp_file.txt', needed by '/home/user/Downloads/UndertaleNDS-0.1/Undertale.nds'.  Stop.
make: *** [Makefile:156: build] Error 2`

**Comment by dant1ux** [comment]
Nevermind, I've solved the problem but I'm not exactly sure how. I just redownloaded the .patch stuff and kept a copy because for some reason the file sizes went to almost nothing after trying to patch and failing. After I put the og .patch files I tried patching (weird name it's more like decrypting) using the patcher.py and running make in different orders I'm not fully sure but I think what I did to make it work were these steps. First was I made sure to create an empty "Undertale.nds" file on the root of the project then I ran "python tools/all.py" and then "python patcher.py" and then finally "make". I hope this helps!
